massdriver-application-azure-vm-docker/main.tf

```diff
@@ -0,0 +1,152 @@
+locals {
+  split_vnet_id        = split("/", var.virtual_network_id)
+  vnet_name            = element(local.split_vnet_id, length(local.split_vnet_id) - 1)
+  vnet_resource_group  = element(local.split_vnet_id, index(local.split_vnet_id, "resourceGroups") + 1)
+  max_length           = 24
+  storage_account_name = substr(replace(var.name, "/[^a-z0-9]/", ""), 0, local.max_length)
+}
+
+module "application" {
+  source              = "github.com/massdriver-cloud/terraform-modules//massdriver-application?ref=ead16fe"
+  name                = var.name
+  service             = "vm"
+  resource_group_name = azurerm_resource_group.main.name
+  location            = azurerm_resource_group.main.location
+}
+
+resource "azurerm_resource_group" "main" {
+  name     = var.name
+  location = var.location
+  tags     = var.tags
+}
+
+resource "azurerm_linux_virtual_machine_scale_set" "main" {
+  name                            = var.name
+  resource_group_name             = azurerm_resource_group.main.name
+  location                        = azurerm_resource_group.main.location
+  disable_password_authentication = false
+  admin_username                  = "placeholder"
```

Comment:
I think we expose this field in postgres and mysql bundles to the customer to select their own password

Reply:
But we don't want people to log into the VM, we won't expose this to the user. 👍